examples/chat/README.md

```diff
@@ -54,7 +55,33 @@ $ weaver single deploy weaver.toml
 $ weaver multi deploy weaver.toml
 ```
 
-## How to run on GKE
+## How to Run on Kubernetes
+
+First, run a MySQL instance in your Kubernetes cluster and initialize it with
+[`chat.sql`](chat.sql). We recommend using [`mysql.yaml`](mysql.yaml) for this:
+
+```shell
+$ kubectl apply -f mysql.yaml
+```
+
+Next, update the `db_uri` field in `weaver.toml` to point to your MySQL
+instance. If you're using `mysql.yaml`, then you should update the `SQLStore`
+section of `weaver.toml` to have the following contents:
```

Comment:
Can you add instructions for updating [kube.repo]?

Reply:
Done.